agent/util-scripts/pbench-move-results

```diff
@@ -216,11 +237,7 @@ while read -r dir; do
     # contain the tarball and the md5 file (as ${tb}.tar.xz.md5.check); pass
     # that information on to pbench-make-result-tb.
     result_dir=$(basename ${dir})
-    if [[ -n "${user}" ]]; then
-        user_arg="--user ${user}"
-    else
-        user_arg=""
-    fi
+    user_arg="--user ${user}"
```

Comment:
Given that `--user` is now required, I would in-line this expression and dispense with the variable.